mon-pix/app/components/authentication/sso-selection-form.gjs

```diff
@@ -37,6 +39,13 @@ export default class SsoSelectionForm extends Component {
     return provider.organizationName;
   }
 
+  get shouldDisplayAccountRecoveryBanner() {
+    const provider = this.oidcIdentityProviders.list?.find((provider) => provider.id === this.selectedProviderId);
```

Comment:
Pas besoin de mettre un opérateur `.?` car `list` va renvoyer un tableau vide si la ressource demandée n’est pas présente cf. https://api.emberjs.com/ember-data/release/classes/store/methods/peekAll?anchor=peekAll

Il y a dans ce fichier 2 lignes où il y a cet opérateur `.?`, mais c’est inutile aussi : 
* `this.oidcIdentityProviders.list?.filter`,
* `this.oidcIdentityProviders.list?.find`,

```suggestion
    const provider = this.oidcIdentityProviders.list.find((provider) => provider.id === this.selectedProviderId);
```